### PR TITLE
feat: Add ability to compose multiple interpolation in a single value.

### PR DESCRIPTION
**Pull request by DanCardin**

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 [tool.poetry]
 name = "configly"
-version = "0.2.2"
+version = "0.3.0"
 description = ""
-authors = []
+authors = ["Dan Cardin <ddcardin@gmail.com>", "Omar Khan <oakhan3@gmail.com>"]
```
Comment (omaralikhn) [inline pyproject.toml:5]: ❤️

**Comment by oakhan3** [comment]
it would be great if docs were added for this, im assuming its to formulate things like uris automatically?

File: tests/test_process.py [hunk 2]
```diff
@@ -135,3 +135,9 @@ def test_object_values(self):
         config = Config(post_process(yaml, input_))
         assert type(config.foo) == Config
         assert config.foo.to_dict() == {"hello": "world"}
+
+    @patch("os.environ", new={"foo": "one", "bar": 'two', 'baz': 'three'})
+    def test_multiple_matches(self):
+        input_ = {"foo": "<% ENV[foo] %>/<% ENV[bar] %>/<% ENV[baz] %>"}
```
Comment (omaralikhn) [inline tests/test_process.py]:
```suggestion
        input_ = {"foo": "<% ENV[foo] %>+<% ENV[bar] %>=<% ENV[baz] %>"}
```

File: tests/test_process.py [hunk 3]
```diff
@@ -135,3 +135,9 @@ def test_object_values(self):
...
+        config = Config(post_process(yaml, input_))
+        assert config.to_dict() == {"foo": "one/two/three"}
```
Comment (omaralikhn) [inline tests/test_process.py]:
```suggestion
        assert config.to_dict() == {"foo": "one+two=three"}
```

**Review by oakhan3 — APPROVED** [review]

**Pull request merged**